import-va-certs.sh

```diff
@@ -8,16 +8,41 @@ set -euo pipefail
     curl -LO https://cacerts.digicert.com/DigiCertTLSRSASHA2562020CA1-1.crt.pem
     curl -LO https://digicert.tbs-certificats.com/DigiCertGlobalG2TLSRSASHA2562020CA1.crt
 
-    # DoD ECA
+    # DoD ECA with multiple fallback mechanisms
     (
-        curl --connect-timeout 10 -LO https://dl.dod.cyber.mil/wp-content/uploads/pki-pke/zip/unclass-certificates_pkcs7_ECA.zip || curl --connect-timeout 10 -LO http://dl.dod.cyber.mil/wp-content/uploads/pki-pke/zip/unclass-certificates_pkcs7_ECA.zip
-        unzip ./unclass-certificates_pkcs7_ECA.zip -d ECA_CA
-        cd ECA_CA/certificates_pkcs7_v5_12_eca/
-        openssl pkcs7 -inform DER -in ./certificates_pkcs7_v5_12_eca_ECA_Root_CA_5_der.p7b -print_certs | awk '/BEGIN/{i++} {print > ("eca_cert" i ".pem")}'
-        rm eca_cert.pem # first one is always invalid because of how awk is breaking it up
-        cp *.pem ../../
+        echo "Downloading DoD ECA certificates..."
+
+        # Primary: HTTPS with timeout and retries
+        if curl --fail --silent --show-error --connect-timeout 10 --max-time 60 --retry 3 --retry-delay 5 -o unclass-certificates_pkcs7_ECA.zip https://dl.dod.cyber.mil/wp-content/uploads/pki-pke/zip/unclass-certificates_pkcs7_ECA.zip; then
+            echo "✓ DoD ECA downloaded via HTTPS"
+        # Fallback 1: HTTP with timeout and retries
+        elif curl --connect-timeout 10 --max-time 60 --retry 3 --retry-delay 5 -LO http://dl.dod.cyber.mil/wp-content/uploads/pki-pke/zip/unclass-certificates_pkcs7_ECA.zip; then
```

Comment:
I recommend removing the elif statement on line 19. The request to http://dl.dod.cyber.mil/wp-content/uploads/pki-pke/zip/unclass-certificates_pkcs7_ECA.zip is automatically forwarded to https://dl.dod.cyber.mil/wp-content/uploads/pki-pke/zip/unclass-certificates_pkcs7_ECA.zip. If the HTTPS request fails, the HTTP request will also fail, making the elif redundant.

Reply:
Oh my bad. I got the two confused.